api/src/services/flavor/flavor.class.js

```diff
@@ -0,0 +1,45 @@
+/* eslint-disable no-unused-vars */
+exports.Flavor = class Flavor {
+  constructor (options, app) {
+    this.options = options || {};
+	this.app = app;
+  }
+
+  async find (params) {
+	let cmd = "SELECT DISTINCT flavor from size";
+	console.log(params);
+	if (params?.query?.product) {
+		cmd += ` WHERE product = '${params.query.product}'`;
+	}
+
+	console.log(cmd);
```

Comment:
This should be removed

Reply:
Oh dear, indeed it should be. I can remove it when I will add the `get` hooks then